test/behavior/generated-parser-behavior.spec.js

```diff
@@ -1483,17 +1498,37 @@ describe("generated parser behavior", function() {
           // Newline representations
           expect(parser).to.failToParse("1\nx", {     // Old Mac
             location: {
+              source: undefined,
               start: { offset: 2, line: 2, column: 1 },
               end: { offset: 3, line: 2, column: 2 }
             }
           });
           expect(parser).to.failToParse("1\r\nx", {   // Windows
             location: {
+              source: undefined,
               start: { offset: 3, line: 2, column: 1 },
               end: { offset: 4, line: 2, column: 2 }
             }
           });
         });
+
+        it("reports location source correctly", function() {
+          const source = { source: "object" };
```

Comment:
Don't bother to change this, but this might have been a cool place to use a Symbol.

Reply:
I think, this is unnecessary -- it is just tests, I've just needed any object only to check that this object is used in the location